source/includes/usage-examples/code-snippets/watch.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+	var uri string
+	if uri = os.Getenv("DRIVER_REF_URI"); uri == "" {
+		log.Fatal("You must set your 'MONGODB_URI' environmental variable. See\n\t https://docs.mongodb.com/drivers/go/current/usage-examples/")
+	}
+
+	client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err = client.Disconnect(context.TODO()); err != nil {
+			panic(err)
+		}
+	}()
+
+	// begin watch
+    coll := client.Database("insertDB").Collection("haikus")
+    pipeline := mongo.Pipeline{bson.D{{"$match", bson.D{{"operationType", "replace"}}}}}
```

Comment:
```suggestion
    pipeline := mongo.Pipeline{bson.D{{"$match", bson.D{{"operationType", "insert"}}}}}
```

Suggestion: Could this use the insert operation instead? I think this is hard to test from compass with "replace" as compass does not have a button for "replace a document", and does have an "insert a document" button (when I was testing this locally I swapped this for insert). This corresponds to changing the below code to reference the Insert One Usage Example.